### PR TITLE
chore: [IA-550] Add Paypal checkout psp info banner

### PR DESCRIPTION
**Pull request by Undermaken**
## Short description
This PR adds a **dedicated banner** to show the psp details (name, costs and privacy url) when a payment is address with Paypal
It also fix about about the total amount shown in the main CTA

### how to test
- do a payment with Paypal

### CODE REVIEW

File: locales/en/index.yml [hunk 1]
```diff
@@ -1029,6 +1029,11 @@ wallet:
         title: "That's it!"
         body: "Now you can continue with your payment."
         primaryButton: "Continue"
+      paymentCheckout:
+        transactionCost: "Costo della transazione"
```
Comment (thisisjp) [inline locales/en/index.yml:1033]:
```suggestion
        transactionCost: "Transaction cost"
```

File: locales/en/index.yml [hunk 2]
```diff
@@ -1029,6 +1029,11 @@ wallet:
...
+        pspHandler: "Gestito da {{pspName}}"
+        privacyDisclaimer: "Proseguendo dichiari di aver letto e compreso i"
```
Comment (thisisjp) [inline locales/en/index.yml:1035]:
```suggestion
        privacyDisclaimer: "By continuing, you declare you have read and understood the"
```

File: locales/en/index.yml [hunk 3]
```diff
@@ -1029,6 +1029,11 @@ wallet:
...
+        privacyTerms: "Termini e condizioni d’uso e l’Informativa Privacy"
```
Comment (thisisjp) [inline locales/en/index.yml:1036]:
```suggestion
        privacyTerms: "Terms and Conditions of Use and Privacy Policy"
```

**Review by thisisjp — CHANGES_REQUESTED** [review]

**Review by thisisjp — APPROVED** [review]

File: ts/features/wallet/paypal/component/PayPalCheckoutPspComponent.tsx [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+    flexDirection: "column"
+  },
+  pspContainer: {
+    alignItems: "flex-start",
+    flexDirection: "column"
+  },
+  pspHeader: {
+    flexDirection: "row"
+  },
+  icon: {
```
Comment (pietro909) [inline ts/features/wallet/paypal/component/PayPalCheckoutPspComponent.tsx:27]: This style seems unused

Comment (Undermaken) [reply ts/features/wallet/paypal/component/PayPalCheckoutPspComponent.tsx]: Yes it does!
Removed here https://github.com/pagopa/io-app/pull/3559/commits/13063366908596b3629fb4b91b3b7232d11bdf43

**Review by pietro909 — APPROVED** [review]

**Pull request merged**